.github/workflows/integrationTests.yaml

```diff
@@ -10,8 +10,52 @@ on:
 
 
 jobs:
+  check:
+    name: Check IT-disabling files
+    outputs:
+      run_job: ${{ steps.check_files.outputs.run_job }}
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: check modified files
+        id: check_files
+        run: |
+          echo "=============== list modified files ==============="
+          git diff --name-only HEAD^ HEAD
+          
+          echo "========== check paths of modified files =========="
+          git diff --name-only HEAD^ HEAD > files.txt
+          while IFS= read -r file
+          do
+            echo $file
+            if [[ $file != generated/* ]] && [[ $file != generator/* ]]; then
+              echo "This modified file is not under the 'generated' nor 'generator' folders."
+              echo "::set-output name=run_job::true"
+            else
+              echo "found a file that belongs in generated/ or generator/"
+              echo "::set-output name=run_job::false"
+              break
```

Comment:
If I am not mistaken, the current logic says if there is at least one file altered belonging to `generat[ed|or]/`, then skip it-tests. As a result, it-test is skipped in your test settings with `generator/dummy-it-trigger.txt` present, even though we also have alterations in this yaml file, which should technically trigger integration tests.

Reply:
btw, from workflow outputs, `set-output` seems to be newly deprecated. 
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Makes sense, I will trigger ITs when at least one file is outside `generat[ed|or]/`